build/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN ATTACHER_TAG=libbpf make build
 
 FROM registry.access.redhat.com/ubi9-minimal:9.2
 RUN microdnf -y update
+RUN yum install -y cpuid
```

Comment:
yum -> microdnf

Reply:
how about limiting to x86_64?
```
RUN if [ $(uname -i) == "x86_64" ]; then microdnf install -y cpuid; fi
```

Reply:
I will try

Reply:
```
Found 0 entitlement certificates
```
either for epel or cpuid there is no package with certificates.

Reply:
as microdnf require a certificates which seems epel don't have. switch the image to ubi and use yum for package install.